### ISSUE TITLE
incrementCounter is incorrect

### ISSUE THREAD

**Issue by defuse** [defuse/php-encryption#120]
From the test case I'm writing:

```
00000000000000000000000000000000 + 257
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'00000000000000000000000000000101'
+'00000000000000000000000000000201'
```

Not good!

**Comment by defuse** [comment]
It's probably because of the `++$inc`. I'm going to re-write it.

**Comment by defuse** [comment]
I fixed this in #116. Close it when that's landed.

**Comment by defuse** [comment]
I rewrote it and it's heavily tested. Closing.
